grpc/weaviate.proto

```diff
@@ -239,10 +247,10 @@ message ResultProperties {
   repeated ReturnRefProperties ref_props = 2;
   string class_name = 3;
   ResultAdditionalProps metadata = 4;
-  repeated FloatArrayProperties floatArrayProperties = 5;
+  repeated NumberArrayProperties numberArrayProperties = 5;
```

Comment:
why here we are not using a snake case for variable names?

Reply:
added a wrong ignore command for the linter. Fixed now